networking/bench/Sender/SenderOptions.hs

```diff
@@ -94,4 +94,4 @@ argsParser =
     recipient = connection >>= \(h, mp) ->
         case mp of
             Just p -> return (fromString h, p)
-            _      -> fail $ "No port specified for host " <> h
+            _      -> _ $ "No port specified for host " <> h
```

Comment:
:cho:

Reply:
I missed this one because I didn't build benchmarks locally.